spark3/src/main/java/org/apache/iceberg/spark/FileScanTaskSetManager.java

```diff
@@ -58,9 +60,19 @@ public void stageTasks(Table table, String setID, List<FileScanTask> tasks) {
     return tasksMap.remove(id);
   }
 
-  private Pair<String, String> toID(Table table, String setID) {
+  public Set<String> fetchSetIDs(Table table) {
```

Comment:
Is this for testing? If so, can we make it package private?

Reply:
Unfortunately this is iceberg.spark and the test checking this is in iceberg.actions

We have this issue where we have two different hierarchies for packaging actions which we may want to resolve latter but this has to be public for now for me to access it in the tests.